integration-tests/tests/src/tests/schema.ts

```diff
@@ -17,34 +17,47 @@
 ////////////////////////////////////////////////////////////////////////////
 
 import { expect } from "chai";
-import { Realm } from "realm";
+import { openRealmBefore } from "../hooks";
+
+interface Test {
+  primary?: number;
+  value?: number;
+}
 
 describe("Realm schema", () => {
   describe("Default property values", () => {
-    it("can take a function as a default property value", () => {
-      interface Test {
-        dynamic?: number;
-      }
-
-      const realm = new Realm({
-        schema: [
-          {
-            name: "Test",
-            properties: {
-              dynamic: {
-                type: "int",
-                default: () => 42,
-              },
+    openRealmBefore({
+      schema: [
+        {
+          name: "Test",
+          primaryKey: "primary",
+          properties: {
+            primary: {
+              type: "int",
+              default: () => 42,
```

Comment:
Can you make this something that returns different values on each call, like an incrementing counter? That would make sure we are actually dynamically calling it each time, rather than evaluating it once to pick a single default.

Reply:
Done